### ISSUE TITLE
Pass data of current row to the FieldTransformer::transform method

### ISSUE THREAD

**Issue by sebheitzmann** [sulu/sulu#6127]
# Problem description

I have a list in the admin with some fields on it. I would like to add a link in one of this field to go on another page. To do that my first idea was to use a transformer to transform the text value in a href link to the other page.

My problem is that the transform fonction receive only two parameter, the field value and the type parameter. 
What i need is the id of the item to link on the right entity.
# Proposed solution

In this function add a third parameter like this : 

```
   renderCells(item: Object): Array<*> {
        const schemaKeys = Object.keys(this.schema);

        return schemaKeys.map((schemaKey, index) => {
            const transformer = listFieldTransformerRegistry.get(this.schema[schemaKey].type);
            const value = transformer.transform(item[schemaKey], this.schema[schemaKey].transformerTypeParameters);

```
```
         const value = transformer.transform(item[schemaKey], this.schema[schemaKey].transformerTypeParameters, item);
```

**Comment by niklasnatter** [comment]
Thanks for creating this issue! For the moment, as a workaround, you can adjust your API to include the information that as required in your transformer in the value of the column.
 
I recently added some thoughts on this topic in https://github.com/sulu/sulu/issues/5918#issuecomment-816696754. The comment also explains the workarounds that are possible at the moment.

**Comment by sebheitzmann** [comment]
Thanks for the tips. I will try that

**Comment by luca-rath** [comment]
Fixed by #6164, will be released as `2.2.12` and `2.3.2`
